content/08.methods.md

```diff
@@ -45,11 +55,44 @@ Drafted sub-sections, edited the manuscript, reviewed pull requests, and coordin
 
 ### Competing interests
 
-A.K. is on the Advisory Board of Deep Genomics Inc.
-E.F. is a full-time employee of GlaxoSmithKline.
-The remaining authors have no competing interests to declare.
-
-The authors last reviewed and approved this competing interests statement on May 26, 2017.
+|Author|Competing Interests|Last Reviewed|
```

Comment:
Date stamping each of these saves us from having to go back to all Version 1.0 authors later for updated conflicts.  Should this information go in `metadata.yaml` instead of the table here?

Reply:
Ooo - I do like building these from datestamps in metadata.yaml!

Reply:
Did this now! Couldn't quite treat as dates due to json output but string form of dates works.